Tools/CISupport/ews-build/steps.py

```diff
@@ -7138,7 +7138,61 @@ class ParseStaticAnalyzerResultsWithoutChange(ParseStaticAnalyzerResults):
     scan_build_output = SCAN_BUILD_OUTPUT_DIR + '-baseline'
 
 
-class FindUnexpectedStaticAnalyzerResults(shell.ShellCommandNewStyle, AddToLogMixin):
+class FindModifiedSaferCPPExpectations(shell.ShellCommandNewStyle, AddToLogMixin):
+    name = 'find-modified-safer-cpp-expectations'
+    RE_FILE = r'^(\+|-)(?P<file>[^/+/-].+(?:\.cpp|\.mm|\.h))$'
+    RE_EXPECTATIONS = r'^(\+\+\+).+(Source/(?P<project>.+)/SaferCPPExpectations/(?P<checker>.+)Expectations)$'
+    command = ['git', 'diff', 'head~1', '--', '*Expectations']
+
+    def __init__(self, **kwargs):
+        super().__init__(logEnviron=False, **kwargs)
```

Comment:
Do you want to add a timeout?

Reply:
I think there is already a default timeout of 1200 seconds